src/cladetime/assign_clades.py

```diff
@@ -60,7 +63,7 @@ def get_sequence_metadata(metadata: pl.DataFrame, sequence_collection_date: date
     ]
 
     # clean and filter metadata (same process used to generate the weekly clade list)
-    filtered_metadata = filter_covid_genome_metadata(metadata, cols)
+    filtered_metadata = get_covid_genome_data(metadata, cols)
```

Comment:
This is a confusing replacement in the context of the names of things and comments.  `get_covid_genome_data` doesn't do the filtering, right?

Reply:
I think this is an artifact of the "review commit by commit" directive--sorry for the confusion! The final version should be clearer!